crates/bevy_winit/src/lib.rs

```diff
@@ -140,6 +140,10 @@ fn change_window(
                         .set_cursor_grab(convert_cursor_grab_mode(grab_mode))
                         .unwrap_or_else(|e| error!("Unable to un/grab cursor: {}", e));
                 }
+                bevy_window::WindowCommand::SetAlwaysOnTop { always_on_top } => {
```

Comment:
This is an extremely minor nit and my bad, but I had just plopped this "wherever" but the rest of the match statement is in the same order as defined in the `WindowCommand` enum.

It would be nice to move it just above `WindowCommand::Close`.

Reply:
done!